components/progress-indicator/__tests__/progress-indicator.browser-test.jsx

```diff
@@ -242,16 +242,6 @@ describe('SLDSProgressIndicator: ', () => {
 				.find('.slds-tooltip-trigger');
 			expect(item).to.have.length(6);
 		});
-
-		it('renders correct assistive text', function () {
-			const item = this.wrapper
-				.find('.slds-progress')
-				.find('.slds-tooltip-trigger')
-				.find('button > span')
-				.find('.slds-assistive-text')
-				.first();
-			expect(item.text()).to.include('custom tooltip #1');
-		});
```

Comment:
Is there a reason this unit test was removed?

Reply:
`.slds-assistive-text` was removed.